src/preferences/dialog/dlgprefsound.cpp

```diff
@@ -88,9 +88,11 @@ DlgPrefSound::DlgPrefSound(QWidget* pParent,
 
     keylockComboBox->clear();
     for (int i = 0; i < EngineBuffer::KEYLOCK_ENGINE_COUNT; ++i) {
-        keylockComboBox->addItem(
-                EngineBuffer::getKeylockEngineName(
-                        static_cast<EngineBuffer::KeylockEngine>(i)));
+        auto engine = static_cast<EngineBuffer::KeylockEngine>(i);
```

Comment:
```suggestion
        const auto engine = static_cast<EngineBuffer::KeylockEngine>(i);
```

Reply:
thanks, cherry-picked this (and same change below in `loadSettings()`).